go.mod

```diff
@@ -1,12 +1,14 @@
 module decred.org/dcrdex
 
-go 1.19
+go 1.21
+
+toolchain go1.21.5
```

Comment:
Same here.  Unless there is a security bug fix that requires forcing everyone who imports the module to download and compile with go1.21.5+, this shouldn't be specified.